### PR TITLE
Added community member profile for Otto Van Der Schaaf

### LINKED ISSUE #1869: [Community] Member Profile: Otto Van Der Schaaf
**Current Behavior**

@oschaaf has been a consistent contributor, community member, and maintainer of Service Mesh Performance.

**Desired Situation**
Let's recognize @oschaaf as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @oschaaf
- Twitter: @oschaaf
- LinkedIn:  https://www.linkedin.com/in/oschaaf
- Link to profile picture: https://avatars.githubusercontent.com/u/2239556?v=4
- Short Bio: interested in proxies, meshes, and performance. Nighthawk maintainer @envoyproxy. PageSpeed (incubating) committer and IPMC member @apache. Principal Envoy Engineer at Red Hat.

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

The CONTRIBUTING.md file has a "Common Types of Site Updates" section with detailed examples to follow here.[CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by hdck007**
Signed-off-by: Haridarshan Choudhary <haridarshanofficial@gmail.com>

**Description**

This PR fixes #1869 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 45f3f7d0a4f28d7528c225e9064490610770ffcb at: https://60a6974dfac1ea7cf0adf230--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 45f3f7d0a4f28d7528c225e9064490610770ffcb at: https://60a6981f76dabf8301970960--layer5ng.netlify.app

File: src/collections/members/otto-van-der-schaaf/index.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+name: Otto Van Der Schaaf
+position: Contributor, Maintainer
+image_path: ./otto-van-der-schaaf.jpg
+github: oschaaf
+twitter: oschaaf
+linkedin: oschaaf
+location: Castricum, North Holland, Netherlands
```
Comment (Nikhil-Ladha) [inline src/collections/members/otto-van-der-schaaf/index.mdx:8]: Let's narrow it down to `North Holland, Netherlands`

**Comment by Nikhil-Ladha** [comment]
Did you fetch the master on this branch?

**Comment by hdck007** [comment]
yeah and then rebased the changes.

**Comment by Nikhil-Ladha** [comment]
And that's the reason things went wrong :).....never fetch the master on the dev branch.
You should have first checked out to the master branch, fetched the latest upstream on it. Merge it with your master then push, then checkout to the dev branch, and then rebase.

**Comment by Nikhil-Ladha** [comment]
Now, you can do one thing but that depends on how the commits were made, i.e, do a git revert to the first commit. Then follow the steps I mentioned above then do make changes and then make a commit.

**Comment by hdck007** [comment]
> And that's the reason things went wrong :).....never fetch the master on the dev branch.
> You should have first checked out to the master branch, fetched the latest upstream on it. Merge it with your master then push, then checkout to the dev branch, and then rebase.

@Nikhil-Ladha please review it now.
Yeah that's exactly what I intended to do but did a slight mistake I guess. Git can be rough man XD.

**Review by Nikhil-Ladha — APPROVED** [review]
Looks good.

**Comment by Nikhil-Ladha** [comment]
@leecalcote I will wait for your signal before merging.

**Comment by l5io** [comment]
🚀 Preview for commit 04468598c5c2ce9ebce3d178260ca25be0f576e3 at: https://60a8f7ac07b42b0f299dbbdf--layer5ng.netlify.app

**Comment by leecalcote** [comment]
Thank you, @hdck007. 

@oschaaf what do you think?

**Review by oschaaf — APPROVED** [review]
Great 👍

**Pull request merged**